presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcClient.java

```diff
@@ -38,6 +38,11 @@ default boolean schemaExists(JdbcIdentity identity, String schema)
         return getSchemaNames(identity).contains(schema);
     }
 
+    default String getIdentifierQuote()
```

Comment:
Let's do not make this a default interface.

Reply:
How should we handle XdbClient (which does not extend BaseJdbcInterface, but presumably it should be refactoed to do so) in fb then? It will never be called but since I added this to the top-level interface it is required to implement it.

Reply:
Actually this looks like a bug in further thought. I think we may need to do conditional logic to disable JdbcPlanOptimizerProvider in the case that the jdbcClient is Xdb